### DISCUSSION TITLE
Add support for headless systems, with Documentation

### DISCUSSION THREAD

**Discussion by DeflateAwning** [siacentral/host-manager#99] [Feature Requests]
I would like to be able to use this management software and dashboard on a Raspberry Pi, where this system runs in a browser instead of in a desktop app. Is this something that could ever happen, or would it require a complete re-work of the existing system?

**Comment by n8mgr** [comment]
It is definitely something that will happen soon. Host Manager v2 is currently being tested by a few insiders and myself. It has support for headless systems through a browser UI, remote management, better optimization, and a whole host of new features.

v1 relies heavily on Electron being present, so it's not really capable of easily converting.

**Comment by DeflateAwning** [comment]
How will such a headless system here differ from this already-existing project? https://github.com/siacentral/sia-host-dashboard

**Comment by n8mgr** [comment]
Host Dashboard does not give you any management capabilities it's just a data display.

**Comment by MarkoPaasila** [comment]
Any progress on this? I suspect most hosters run headless systems.

**Comment by n8mgr** [comment]
> Any progress on this? I suspect most hosters run headless systems.

Due to limited bandwidth there have been some delays, but I’ve been making steady progress. I don’t have a definitive timeline, but a public beta should be available soon.

**Comment by MarkoPaasila** [comment]
I'll just leave this here to help other noobs like me. The command to forward remote ports to localhost before starting host-manager is `ssh -L 9980:localhost:9980 -L 9981:localhost:9981 -L 9982:localhost:9982 -L 9983:localhost:9983 -L 9984:localhost:9984 sia-host.lan`

**Comment by n8mgr** [comment]
> I'll just leave this here to help other noobs like me. The command to forward remote ports to localhost before starting host-manager is `ssh -L 9980:localhost:9980 -L 9981:localhost:9981 -L 9982:localhost:9982 -L 9983:localhost:9983 -L 9984:localhost:9984 sia-host.lan`

You only need to forward `9980`

**Comment by ghost** [comment]
> It is definitely something that will happen soon. Host Manager v2 is currently being tested by a few insiders and myself. It has support for headless systems through a browser UI, remote management, better optimization, and a whole host of new features.
> 
> v1 relies heavily on Electron being present, so it's not really capable of easily converting.

I'm very excited about this!
